std_msgs/Header header
SimpleChange[] added
SimpleChange[] removed
SimpleChange[] preserved
MoveChange[] moved

